tests/support_server/setup.pm

```diff
@@ -55,9 +55,9 @@ sub setup_pxe_server {
 sub setup_http_server {
     return if $http_server_set;
 
-    $setup_script .= "rcapache2 stop\n";
+    $setup_script .= "systemctl stop apache2\n";
```

Comment:
at best this variable should be deleted at all , but this requires much bigger effort ...  
but at least you make it better ...

Reply:
You're right, maybe I will try this in the next modification :) I keep this in my mind.